bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -122,6 +122,20 @@ trigf = h5py.File(args.trigger_file, 'r')
 logging.info('Opening template file: %s' % args.bank_file)
 templatef = h5py.File(args.bank_file, 'r')
 
+# Count the number of triggers in each template, for calulation of the true noise rate
+logging.info('Counting number of triggers in each template')
+tb = trigf[args.ifo+'/template_boundaries'][:]
+hash_sort = np.argsort(templatef['template_hash'][:])
+tb_sorted = tb[hash_sort]
+# calculate the differences between the boundary indices, 
+# in order to get the number in each template
+count_in_template = tb_sorted[1:] - tb_sorted[:-1]
```

Comment:
can you use numpy.diff() for this?

Reply:
yes - I had just used this form as it is what was used elsewhere (https://github.com/gwastro/pycbc/blob/8fe2b5ed427a33b74ca862de85acee5336c323f7/bin/hdfcoinc/pycbc_coinc_mergetrigs#L140). I will change to diff though